tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/observability/metrics/WebSocketMetricsTestCase.java

```diff
@@ -82,24 +84,30 @@ private void setup() throws Exception {
      */
     @Test
     public void testMetrics() throws Exception {
-        WebSocketTestClient client = new WebSocketTestClient("ws://localhost:9090/basic/ws");
-        client.handshake();
-        client.sendText(MESSAGE);
-        client.sendText(MESSAGE);
-        client.sendText(MESSAGE);
-        client.sendText(MESSAGE);
-        client.sendText(MESSAGE);
-        client.sendPing(SENDING_BYTE_BUFFER);
-        client.sendPong(SENDING_BYTE_BUFFER);
-        client.sendBinary(SENDING_BYTE_BUFFER);
-        client.sendText(CLOSE_MESSAGE);
+        // Test Service
+        final WebSocketTestClient[] client = new WebSocketTestClient[1];
+        await().atMost(20, TimeUnit.SECONDS)
```

Comment:
Why we can't use setCountDownLatch()[https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/util/websocket/client/WebSocketTestClient.java#L101]  instead of this?